server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQWebAdminServerIntegrationTest.java

```diff
@@ -198,6 +198,39 @@ void solveMailboxInconsistenciesTaskShouldBeExposed() {
             .body("additionalInformation.conflictingEntries", hasSize(0));
     }
 
+    @Test
+    void solveMessageInconsistenciesTasksShouldBeExposed() {
+        String taskId = with().post(UPGRADE_TO_LATEST_VERSION)
+            .jsonPath()
+            .get("taskId");
+
+        with()
+            .get("/tasks/" + taskId + "/await")
+            .then()
```

Comment:
usually we indent `when()`, `.then()` etc a step back, to have a better visibility with those different steps